src/compiler/types.ts

```diff
@@ -9936,6 +9936,7 @@ export interface UserPreferences {
     readonly organizeImportsNumericCollation?: boolean;
     readonly organizeImportsAccentCollation?: boolean;
     readonly organizeImportsCaseFirst?: "upper" | "lower" | false;
+    readonly excludeLibrarySymbols?: boolean;

```

Comment:
I think the name of this setting needs to mention the command/feature it affects.